src/start/PasswordAuthView.js

```diff
@@ -43,7 +45,9 @@ class PasswordAuthView extends React.Component {
       progress: false,
       email: props.email || config.defaultLoginEmail,
       password: config.defaultLoginPassword,
-      error: ''
+      error: '',
+      ishidden: true,
+      toggleText: 'Show',
```

Comment:
You don't have to use both of these. You can determine the text from the 'isHidden' value.